### ISSUE TITLE
BUG - sev 3 - Android - Accessibility - attach a photo help, screen reader reading text that is offscreen

### ISSUE THREAD

**Issue by DJUltraTom** [department-of-veterans-affairs/va-mobile-app#5062]
## What happened?

When reading the help file for attaching a document to a secure message using a screen reader, if I scroll down on the text, the screen reader focus doesn't always move to what it is on the screen.

## Specs:

- Device: google pixel 4a
- OS: android
- User Account:
- Accessibility State: enabled

## Steps to Reproduce

- using talkback, navigate to secure messages and tap "how to attach a file"
- scroll down to the bottom of the text on the help screen
- swipe left and observe that the focus moves up the screen while reading content but it is reading behind the header and is not auto scrolling

## Desired behavior

focus should move and read items on the page

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Issue Tracking](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/VA%20Mobile%20App%20Test%20Plan.md#issue-tracking) for details on Severity Levels

sev 3

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, feature, bug)
- [ ] Estimate of 1 added

**Comment by TKDickson** [comment]
changing from global to health, as the camera is only used in health (messaging) and benefits (claims file request) features.

**Comment by alexandec** [comment]
Can confirm that this issue is still present as of May 2024.

**Comment by Sparowhawk** [comment]
Unfortunately unless we want to change the display of this screen there is nothing we can do. Talkback overrides us from consuming swipe gestures so we can't even tell when the user swipes to manually scroll the screen for the user.

This is a Android system related issue that it doesn't scroll properly with scrollviews with talkback. The recommended solution I found was to reduce the amount of content on the screen so it isn't an issue.

**Comment by Sparowhawk** [comment]
@bischoffa not sure who has the final say on this, but unfortunately this bug isn't fixable unless we reduce the amount of content on this screen so it isn't an issue(would still be an issue with large font sizes).

So my recommendation is that we can close this as non fixable and just hope that the android OS fixes this eventually.

**Comment by TKDickson** [comment]
As a H&B bug, either @rbontrager and @ala-yna should agree with the decision to close as non-fixable, and then close it.

**Comment by Sparowhawk** [comment]
https://stackoverflow.com/questions/57514973/detect-swipe-left-and-right-on-page-level

The first comment mentions how we cannot consume swipe gestures due to voice over/talkback intercepting them.

**Comment by rbontrager** [comment]
@ala-yna Yep I agree.  Closing as a non-fixable android issue for now due to Dylan's comment above.
